Client/Application/WebServer.swift

```diff
@@ -18,7 +18,7 @@ class WebServer {
     }
 
     func start() -> Bool {
-        return server.running || server.startWithPort(0, bonjourName: nil)
+        return server.running || server.startWithOptions([GCDWebServerOption_Port: 0, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true], error: nil)
```

Comment:
What's the `GCDWebServerOption_BindToLocalhost` change for?

Also, if I understand the fix correctly, the `GCDWebServerOption_AutomaticallySuspendInBackground` change is unnecessary now, right? That is, if we don't save port-specific URLs anymore, it shouldn't matter if our server port changes.